src/client/command.cc

```diff
@@ -97,9 +97,8 @@ bool Command::GenerateFromArgs(int argc, const char* const raw_argv[],
     // TODO: move this code inside |DriverCommand| constructor.
     if ((command.getSource().getKind() != Action::AssembleJobClass &&
          command.getSource().getKind() != Action::BackendJobClass &&
-         command.getSource().getKind() != Action::CompileJobClass &&
-         command.getSource().getKind() != Action::PrecompileJobClass &&
```

Comment:
As those work wrong on remotes. It looks like a PCH file generated on remote host doesn't work locally. There're may be several different reasons for this:
1. OS/Arch differencies(by order?) as pch/pth files are somethig like a memory dump.
2. These files contain absolute paths to source header.